### DISCUSSION TITLE
Welcome to tidy-ui Discussions!

### DISCUSSION THREAD

**Discussion by badatt** [badatt/tidy-ui#139] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
Hi @badatt.

**Reply by badatt** [reply]
Hello @raphaellouisandress

**Reply by ghost** [reply]
Hi @badatt, how are you?

**Reply by badatt** [reply]
Not bad, how are you @raphaellouisandress

**Reply by ghost** [reply]
Hi @badatt, I'm fine. 

**Note:** I think this open source project is amazing, and I hope to contribute with ui/ux. What does it take to help this open source project: tidy-ui? do you have any guide for contributor?

**Reply by badatt** [reply]
I didn't think about contributions yet, but I am happy to collaborate with the community.
As of now, I don't have any specific UI/UX designs ready :smile:, just started building with a pattern and continued in that flow. 
I welcome your suggestions, do let me know

**Reply by ghost** [reply]
Hi @badatt.

> I welcome your suggestions, do let me know

can we have tidy-ui for react-js, react-native?

**Reply by badatt** [reply]
Currently, tidy-ui supports only react-js, I don't have any experience with react-native. How much impact does it make if we bring react-native support to tidy-ui?

**Reply by ghost** [reply]
> Currently, tidy-ui supports only react-js, I don't have any experience with react-native.

It is very common nowadays to create mobile apps with react-native. As the base code: tidy-ui  is made in react, it would be easy to port in react-native.

> How much impact does it make if we bring react-native support to tidy-ui?

In a way the impact would be to increase the development of the graphical interface on mobile devices. In other words, increase the number of use cases or 'showcase' of the tidy-ui.  What do you think of this idea?

**Reply by badatt** [reply]
> What do you think of this idea?

The idea sounds great, let's start something about support for react-native and move things slowly. That being said, my priority is web users for now.

**Reply by ghost** [reply]
Hi @badatt,

> The idea sounds great, let's start something about support for react-native and move things slowly. That being said, my priority is web users for now.

thanks for all information again, and feedback here.
